targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8C63XX/TARGET_MCU_PSOC6_M0/device/TOOLCHAIN_IAR/cy8c6xx7_cm0plus.icf

```diff
@@ -0,0 +1,218 @@
+/***************************************************************************//**
```

Comment:
I'll add a comment here as I can't comment in the above .a  files.

`https://github.com/ARMmbed/mbed-os/tree/16ee057d462219ac9b7069fde1021c06946119df/targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8C63XX/TARGET_MCU_PSOC6_M0/device` this folder should contain readme + license (PBL preferably) for binaries there. 

Readme should address: What binaries are there, what tools versions they were compiled with, and all these binaries are I assume released with the same license (there will be separate license file provided).

Reply:
Is it ok to just reference PBL on the ARM site in the readme file? Or should I include license file with PBL text copied in?

Reply:
Just add a license as well please (License.txt). It's already present in other similar binaries in the tree.